src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheckTest.java

```diff
@@ -81,13 +91,44 @@ public void testIncorrect() throws Exception {
         verify(checkConfig, getPath("InputSummaryJavadocIncorrect.java"), expected);
     }
 
+    @Test
+    public void testInlineIncorrect() throws Exception {
+        final DefaultConfiguration checkConfig = createModuleConfig(SummaryJavadocCheck.class);
+        checkConfig.addAttribute("forbiddenSummaryFragments",
+                "^@return the *|^This method returns |^A [{]@code [a-zA-Z0-9]+[}]( is a )");
+        final String[] expected = {
+            "13: " + getCheckMessage(MSG_SUMMARY_JAVADOC_MISSING),
+            "24: " + getCheckMessage(MSG_SUMMARY_JAVADOC),
+            "29: " + getCheckMessage(MSG_SUMMARY_FIRST_SENTENCE),
+            "34: " + getCheckMessage(MSG_SUMMARY_FIRST_SENTENCE),
+            "44: " + getCheckMessage(MSG_SUMMARY_FIRST_SENTENCE),
+            "49: " + getCheckMessage(MSG_SUMMARY_FIRST_SENTENCE),
+            "54: " + getCheckMessage(MSG_SUMMARY_JAVADOC_MISSING),
+            "77: " + getCheckMessage(MSG_SUMMARY_FIRST_SENTENCE),
+            "98: " + getCheckMessage(MSG_SUMMARY_JAVADOC),
+            "109: " + getCheckMessage(MSG_SUMMARY_JAVADOC_MISSING),
+            "144: " + getCheckMessage(MSG_SUMMARY_FIRST_SENTENCE),
+            "149: " + getCheckMessage(MSG_SUMMARY_FIRST_SENTENCE),
+            "154: " + getCheckMessage(MSG_SUMMARY_JAVADOC_MISSING),
+            "165: " + getCheckMessage(MSG_SUMMARY_FIRST_SENTENCE),
+            "183: " + getCheckMessage(MSG_SUMMARY_JAVADOC_MISSING),
+            "188: " + getCheckMessage(MSG_SUMMARY_JAVADOC_MISSING),
+            "193: " + getCheckMessage(MSG_SUMMARY_JAVADOC_MISSING),
+            "198: " + getCheckMessage(MSG_SUMMARY_JAVADOC_MISSING),
+            "203: " + getCheckMessage(MSG_SUMMARY_JAVADOC_MISSING),
+            "208: " + getCheckMessage(MSG_SUMMARY_JAVADOC_MISSING),
+        };
+        verify(checkConfig, getPath("InputSummaryJavadocInlineIncorrect.java"), expected);
```

Comment:
Input file can be used only for one config (one test method), if you need same Input file content, you do copy of it, and there will be required modifications in trailing comments as new config most likely make amount/place of violations different in file.

Reply:
Done